editors/code/src/highlighting.ts

```diff
@@ -253,14 +253,14 @@ function randomU32Numbers(seed: number) {
         random ^= random >> 17;
         random ^= random << 5;
         random |= 0;
-        return random
-    }
+        return random;
+    };
 }
 
 function hashString(str: string): number {
     let res = 0;
     for (let i = 0; i < str.length; ++i) {
-        const c = str.codePointAt(i)!!;
```

Comment:
You are too assertive here, one exclamation would be enough

Reply:
Probably a leftover of working in kotlin, where it is always double exclamation mark to assert not null